huntflow_api_client/models/response/dictionaries.py

```diff
@@ -0,0 +1,70 @@
+from typing import List, Optional
+from uuid import UUID
+
+from pydantic import BaseModel, Field, PositiveInt
+
+from huntflow_api_client.models.utils import descriptions
+from huntflow_api_client.models.utils.fields import DatetimeWithTZ
+
+
+class DictionaryTaskResponsePayload(BaseModel):
+    task_id: UUID = Field(..., description="Job ID")
+
+
+class DictionaryTaskResponseMeta(BaseModel):
+    data: dict = Field(..., description="Request body")
+    account_id: int = Field(..., description=descriptions.organization_id)
+
+
+class DictionaryTaskResponse(BaseModel):
+    status: str = Field(..., description="Operation status", example="ok")
+    payload: DictionaryTaskResponsePayload
+    meta: DictionaryTaskResponseMeta
+
+
+class DictionaryUpdateResponse(DictionaryTaskResponse):
+    pass
+
+
+class DictionaryCreateResponse(DictionaryTaskResponse):
+    pass
+
+
+class DictionaryItem(BaseModel):
+    id: PositiveInt = Field(..., description=descriptions.dict_id, example=7)  # noqa: A003 VNE003
+    code: str = Field(..., description=descriptions.dict_code, example="citizenship")
+    name: str = Field(..., description=descriptions.dict_name, example="Citizenship")
+    foreign: Optional[str] = Field(None, description=descriptions.foreign, example="d_ctz")
+    created: DatetimeWithTZ = Field(..., description="Date and time of creating a dictionary")
```

Comment:
created: **datetime** = Field(..., description="Date and time of creating a dictionary")